CefSharp.BrowserSubprocess.Core/JavascriptObjectWrapper.cpp

```diff
@@ -40,13 +52,59 @@ namespace CefSharp
 			wrappedproperty->V8Value = javascriptObject;
 			wrappedproperty->Bind();
 
-			_wrappedProperties->Add(wrappedproperty);
+            _wrappedProperties->Add(prop->JavascriptName, wrappedproperty);
 		}
 	}
 
 	BrowserProcessResponse^ JavascriptObjectWrapper::GetProperty(String^ memberName)
 	{
-		return _browserProcess->GetProperty(_object->Id, memberName);
+		auto resp = _browserProcess->GetProperty(_object->Id, memberName);
```

Comment:
General formatting including, spaces, braces on new lines. More expressive variable naming. 
I won't point out each case, I'll let you clean it up as a whole.